gensim/models/wrappers/ldamallet.py

```diff
@@ -245,11 +247,29 @@ def show_topic(self, topicid, topn=10):
     def print_topic(self, topicid, topn=10):
         return ' + '.join(['%.3f*%s' % v for v in self.show_topic(topicid, topn)])
 
+
+    def get_version(self, direc_path):
+        """"
+
+        function to return the version of `mallet`
+
+        """
+        xml_path = direc_path.split("bin")[0]
+        try:
+            doc = et.parse(xml_path+"pom.xml").getroot()
```

Comment:
Code style: put a space around binary operators, here and elsewhere in this PR (PEP8).